src/AppBundle/Controller/FeedbackController.php

```diff
@@ -0,0 +1,70 @@
+<?php
+namespace AppBundle\Controller;
+
+use Symfony\Component\HttpFoundation\Request;
+use AppBundle\Form\Type\FeedbackType;
+use AppBundle\Entity\Feedback;
+use AppBundle\Service\SlackMessenger;
+
+class FeedbackController extends BaseController
+{
+    //shows form for submitting a new feedback
+    public function indexAction(Request $request)
+    {
+        $feedback = new Feedback;
+        $user = $this->getUser();
+        $form = $this->createForm(FeedBackType::class, $feedback);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            //Stores the submitted feedback
+            $em = $this->getDoctrine()->getManager();
+            $feedback = $form->getData();
+            $feedback->setUser($user);
+            $em->persist($feedback);
+            $em->flush();
+
+            //Notifies on slack (NotificationCHannel)
+            $messenger = $this->container->get('AppBundle\Service\SlackMessenger');
+            $messenger->notify($feedback->getSlackMessageBody());
+
+            return $this->redirect($request->getUri()); //Makes sure the user cannot submit the same form twice (e.g. by reloading page)
```

Comment:
Tips: Ved å legge til en flashmelding vil brukeren få opp en melding øverst på siden etter redirect, feks: `$this->addFlash("success", "Tilbakemeldingen har blitt registrert, tusen takk");`.

"success" = grønn,
"warning" = oransje,
"danger" = rød